From a89f1e0d060e3690591ba441fa0b322260007c14 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 26 Feb 2019 16:59:49 +0100
Subject: [PATCH] Fix broken translations in all libhandy applications

Translations in all the applications using libhandy would be broken
after a call to hdy_init() as it changed the default gettext translation
domain.

See https://gitlab.gnome.org/GNOME/gnome-control-center/issues/393

Applied-Upstream: in upstream libhandy, g-c-c didn't pull the new copy
into its submodule at the time of writing
---
 src/hdy-main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/subprojects/libhandy/src/hdy-main.c b/src/hdy-main.c
index 08ef1e3..d6e71da 100644
--- a/subprojects/libhandy/src/hdy-main.c
+++ b/subprojects/libhandy/src/hdy-main.c
@@ -42,7 +42,6 @@ hdy_init (int *argc, char ***argv)
   if (hdy_initialized)
     return TRUE;
 
-  textdomain (GETTEXT_PACKAGE);
   bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
   bindtextdomain (GETTEXT_PACKAGE, LOCALEDIR);
   g_resources_register (hdy_get_resource ());
-- 
2.20.1

